geode-core/src/main/java/org/apache/geode/management/DiskStoreMXBean.java

```diff
@@ -191,6 +191,11 @@
    */
   float getDiskUsageCriticalPercentage();
 
+  /**
+   * Returns the usage percentage of the disk storage
+   */
+  public int getDiskUsagePercentage();
```

Comment:
The `public` modifier is not necessary here. Also, see comment about using `float` instead of `int`.